service-locator/core/src/main/resources/reference.conf

```diff
@@ -29,6 +29,22 @@ lagom.akka.discovery {
     https = https
   }
 
+  # A mapping of service names to lookup information. Each mapping should define the following:
+  # - service-name - The name of the service. If undefined, will use the same name.
+  # - port-name - The name of the port, for example, http. If undefined, will use lagom.akka.discovery.defaults.port-name
+  # - port-protocol - The protocol. If undefined, will use lagom.akka.discovery.defaults.port-protocol
+  # - scheme - The scheme to return in the URI. If undefined, will attempt to map the port-name to a scheme using
+  #   lagom.akka.discovery.port-name-scheme-mapping, and if that fails, will use lagom.akka.discover.defaults.scheme.
+  # For example:
+  # service-name-mappings {
+  #   my-service-name {
+  #     service-name = my-service.default.svc.cluster.local
```

Comment:
Do we still need `service-name`? 

It seems that this just an alternative way of `service-name` + `service-name-suffix`. 

Moreover, `service-name-suffix` can be removed because https://github.com/akka/akka/issues/25825 is merged.